### PR TITLE
feature/numbers: percentage and currency

### PR DESCRIPTION
**Pull request by billyxs**
number like things doing number like stuff

### CODE REVIEW

File: packages/currency/__tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React from 'react'
```
Comment (seesleestak) [inline packages/currency/__tests__/index.test.js]: Nice tests

File: packages/hui-pages/src/containers/GlobalStylesInjector.js [hunk 2]
```diff
@@ -82,6 +82,10 @@ body {
 *:after {
   box-sizing: inherit;
 }
+
+a {
+  text-decoration: none;
+}
```
Comment (seesleestak) [inline packages/hui-pages/src/containers/GlobalStylesInjector.js]: Is this to override the `<Link />` styles?

File: packages/hui-pages/src/routes/PercentageDocs/index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Percentage from '@hixme-ui/percentage'
+
+import { Code, DocsTitle } from '../../components'
+
+const CurrencyDocs = () => (
+  <div>
+    <DocsTitle title='Percentage' subtitle='A percentage component providing formatting for percent like things' />
```
Comment (seesleestak) [inline packages/hui-pages/src/routes/PercentageDocs/index.js]: You are quite the wordsmith

File: packages/hui-pages/src/containers/GlobalStylesInjector.js (hunk 2 above)
Comment (billyxs) [reply packages/hui-pages/src/containers/GlobalStylesInjector.js]: It's to remove underlines that we've never had on links.  It's the default style for links, not <Link /> specifically

File: packages/hui-pages/src/routes/PercentageDocs/index.js (hunk 3 above)
Comment (billyxs) [reply packages/hui-pages/src/routes/PercentageDocs/index.js]: Who want's to read boring docs?

File: packages/hui-pages/src/containers/GlobalStylesInjector.js (hunk 2 above)
Comment (seesleestak) [reply packages/hui-pages/src/containers/GlobalStylesInjector.js]: Cool

**Pull request merged**